### PR TITLE
Cleanup tech debt

### PR DESCRIPTION
**Pull request by vladdoster**
Cleaned up:
- README
- `test_bash.sh`

Added:
- logic to remove linter files if user is not using that linter
- flake8 dotfile in project which did not exist

### CODE REVIEW

File: hooks/post_gen_project.sh [hunk 1]
```diff
@@ -1,47 +0,0 @@
-#!/usr/bin/env bash
-
-if ! make venv; then
```
Comment (patdaburu) [inline hooks/post_gen_project.sh:3]: Wow, Vlad!  Many thanks for all your work here!!  I am wondering though why you removed the secondary initialization stuff (creating the virtual environment, installing the requirements, and calling build).  Does it cause a problem?

File: hooks/post_gen_project.sh [hunk 2]
```diff
@@ -1,47 +0,0 @@
...
-                 _       _
-                | |     | |
-              __| | __ _| |__  _   _ _ __ _   _
-             / _` |/ _` | '_ \| | | | '__| | | |
-            | (_| | (_| | |_) | |_| | |  | |_| |
-             \__,_|\__,_|_.__/ \__,_|_|   \__,_|
-
-LOGO
-
-echo -e "\033[1m\033[4mNext Steps\033[0m"
```
Comment (patdaburu) [inline hooks/post_gen_project.sh:44]: Same as above:  It seems like the "next step" messages are helpful to someone not familiar with the structure.  Thoughts?  (I have no problem removing the ASCII art if it seems too gaudy.)

**Review by patdaburu — COMMENTED** [review]
Many, many thanks for your work here, Vlad.  I just have a couple of questions in the comments.

File: hooks/post_gen_project.sh (hunk 1 above)
Comment (vladdoster) [reply hooks/post_gen_project.sh:3]: Not a problem, but should be left up to user. Maybe they don't want a venv or want it elsewhere. It should really be left up to the user.

File: hooks/post_gen_project.sh (hunk 2 above)
Comment (vladdoster) [reply hooks/post_gen_project.sh:44]: ### Ascii art

Not gaudy so much as just not really useful. I can add back the `daburu`, but the `python` isnt useful and just clogs up the terminal for real upside.

Comment (vladdoster) [reply hooks/post_gen_project.sh:44]: Let me add more precise usage docs and this change will be easier to stomach.

Comment (patdaburu) [reply hooks/post_gen_project.sh:44]: No need to re-introduce the ASCII art.  Anything that's not helpful can go.

**Review by patdaburu — APPROVED** [review]
One more time:  6.02*10^23 thanks for everything, Vlad!!  You're best!!

**Pull request merged**